src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/TypeBuilderImpl.cs

```diff
@@ -734,7 +734,7 @@ private static bool MatchesTheFilter(MethodBuilderImpl method, BindingFlags meth
 
                 if (argType.IsArray != paramType.IsArray ||
                     argType.IsByRef != paramType.IsByRef ||
-                    argType.IsPointer != argType.IsPointer)
+                    argType.IsPointer != paramType.IsPointer)
```

Comment:
```suggestion
                    argType.IsPointer != argType.IsPointer)
```
I am going to deal with this one separately